lib/9pfs/9pfs_vnops.c

```diff
@@ -1031,6 +1031,64 @@ static int uk_9pfs_ioctl(struct vnode *dvp, struct vfscore_file *fp,
 	 */
 	case FIONBIO:
 		return 0;
+	/* For every `ioctl` request related to a terminal, return ENOTTY. */
```

Comment:
Are you sure these are all? What about `TIOCGLCKTRMIOS` or `TIOCSLCKTRMIOS`? :thinking:

Reply:
Indeed, missed a few. Should be fine now :pray: